functions_and_scope_lab.js

```diff
@@ -0,0 +1,289 @@
+// # Function Exercises
+//
+console.log(`\n## Question 1`);
+//
+// Write a function that returns the largest of two numbers. If they are equal, return the second one.
+//
+// Example:
+//
+// Input: `12, 4`
+//
+// Output: `12`
+const largeNum = (input1,input2) => {
+    if (input1 > input2){
+      return input1
+        } else if (input1 < input2) {
+            return input2
+          } else {
+                return input2
+            }
+}
+
+let numResult = largeNum(4,9);
+console.log(numResult);
+
+
+console.log(`\n## Question 2`);
+//
+// Write a function named getDogAge that takes one argument: a dog's age in human years. The function returns the dog's age in dog years. The conversion rate is 1 human year to 7 dog years.
+//
+// Example:
+//
+// Input: `4`
+//
+// Output: `28`
+
+const getDogAge = (ageInHuman) => {
+  let ageInDog = 1;
```

Comment:
You could just return ageInHuman * 7 instead of creating new variable inside called ageInDog. Still same result though